upstream-configuration/composer.json

```diff
@@ -44,7 +44,17 @@
         "bower-asset/fontawesome": "5.15.3",
         "drupal/current_page_crumb": "^1.3",
         "drupal/pathauto": "^1.8",
-        "asuwebplatforms/webspark-module-webspark_isearch": "dev-main"
+        "asuwebplatforms/webspark-module-webspark_isearch": "dev-main",
+        "drupal/media_library_form_element": "^2.0",
+        "drupal/admin_toolbar": "^3.0",
```

Comment:
Just a heads-up that the ASU Brand header will probably not play well with the drop downs from the admin_toolbar menu. Probably just a Z-index override that needs to be added somewhere in the theme.